src/VisualStudio/IntegrationTest/TestUtilities/InProcess/Editor_InProc_NavigationBar.cs

```diff
@@ -81,6 +87,28 @@ private List<ComboBox> GetNavigationBarComboBoxes(IWpfTextView textView)
 
         private static UIElement GetNavbar(IWpfTextView textView)
         {
+            // Visual Studio 2019
```

Comment:
I admit seeing this comment I thought this was only for the first two lines, with the "Make sure we have the top pane..." line was a different section.

Reply:
Agreed. All the 2017 code can go away once 16.0 is released and integration tests work there.